specification/cognitiveservices/data-plane/ContentSafety/readme.md

```diff
@@ -0,0 +1,78 @@
+# Content Safety
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Content Safety.
+
+## Getting Started
+
+To build the SDKs for My API, simply install AutoRest via `npm` (`npm install -g autorest`) and then run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the Content Safety.
+
+```yaml
+openapi-type: data-plane
+tag: package-2023-04-30-preview
+```
+
+### Tag: package-2023-04-30-preview
+
+These settings apply only when `--tag=package-2023-04-30-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2023-04-30-preview'
+input-file:
+  - preview/2023-04-30-preview/contentsafety.json
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+```yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python-track2
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-go
```

Comment:
Please remove the go lib. That CI would block merge.

Reply:
Removed azure-sdk-for-go and added azure-sdk-for-net

Reply:
nit, I assume .NET would be azure-sdk-for-net-track2